scripts/variantstore/docs/aou/vds/Creating a VDS.md

```diff
@@ -3,12 +3,28 @@
 **NOTE:** This doc is very much a work in process and is serving as a way to keep track of Hail commands that were used for Delta after the initial VDS delivery.
 This example walks through the procedure we followed for creating a VDS.
 
+## Create avro files for VDS extract
+Run GvsExtractAvroFilesForHail to create a directory of avro files that will then be used to create a VDS
+
 ## Configuring the Terra notebook cluster
 
 Please follow the instructions
 in [AoU Delta VDS Cluster Configuration](cluster/AoU%20Delta%20VDS%20Cluster%20Configuration.md) to set up a Delta-scale
 cluster.
 
+## Using the created notebook, run the python script to create a VDS from the avro files created by the GvsExtractAvroFilesForHail WDL
+These avro files will be passed in using the --avro-path parameter
+The vds-path parameter tells the WDL where to write the final desired VDS to
+There is quite a bit of temporary data created and while it will be cleaned up later, it's helpful to hand the python script a location.
```

Comment:
is it 'helpful' or is it required? I assume you are talking about the `temp-path` directory? (so clarify this line).

Reply:
I think it's only needed if there is a lot of data, but better to require it when it's not needed then the opposite